custom_components/grott/sensors/sensors_mqtt.py

```diff
@@ -1469,6 +1469,16 @@ def datetime_formatter(mqtt_data):
     "func": lambda js: js['values']["BatWatt"],
     "divider": 10,
     "unique_name": "mqtt_135",
+  }, 
```

Comment:
Could you please make this match the block up on line 688, there's no reason they can't have the same names.
e.g.
```
{
    "name": "Import from Grid Power",
     "device_class": SensorDeviceClass.POWER,
     "unit_of_measurement": UnitOfPower.WATT,
     "state_class": SensorStateClass.MEASUREMENT,
     "icon": "mdi:home-import-outline",
     "func": lambda js: js['values']["pactouserr"],
     "divider": 10,
     "unique_name": "mqtt_051",
},
```

Obviously the unique name etc. should remain as you've put them. but the name and icon etc. should match the other one, just for consistency.

Reply:
done, thanks for the feedback.